code/modules/clothing/glasses/glasses.dm

```diff
@@ -21,7 +21,7 @@
 	desc = "nothing."
 	icon_state = "purple"
 	item_state = "glasses"
-	origin_tech = Tc_MATERIALS + "=1"
+	origin_tech = Tc_MATERIALS + "=1;" + Tc_ANOMALY + "=1"
```

Comment:
You don't seem to have said anything about this

Reply:
Bottom on the OP. But not in chabgelog.

Reply:
Why wouldn't this be in the changelog?

Reply:
It was something I thought was too small to put into the change log. I will put it in there to calm people's autism.

Reply:
hey so fuck off with this